src/js/routes/result/Result.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import BackButton from '../../components/BackButton.jsx';
+
+const Result = (props) => (
```

Comment:
What is the intended purpose of this? For showing a simple page given a particular item search? I think I should get the ball rolling for an actual design of this project. As it is I don't see the point of this if it doesn't show anything more about an item than what is shown in the table we intend to display.